### PR TITLE
add russian

### PR DESCRIPTION
**Pull request by JohnJ255**

### CODE REVIEW

File: tests/test_text_to_num_ru.py [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+    def test_examples(self):
+        source = "пятьдесят один миллион пятьсот семьдесят восемь тысяч триста два"
+        expected = 51578302
+        self.assertEqual(expected, text2num(source, 'ru'))
+
+        source = "восемьдесят один"
+        expected = 81
+        self.assertEqual(expected, text2num(source, 'ru'))
+
+    def test_sets_of_ones(self):
```
Comment (rtxm) [inline tests/test_text_to_num_ru.py:189]: Can you explain this test please?

**Review by rtxm — COMMENTED** [review]
Thank you very much for your contribution Eugene! I'll take a more in-depth look into it when I have more time.

File: tests/test_text_to_num_ru.py (hunk 1 above)
Comment (JohnJ255) [reply tests/test_text_to_num_ru.py:218]: it is test for all cases for hundred, thousand, million etc with similar letters and words
may be this is an extra test now, but it was helpful on the one of first steps of developing

Comment (JohnJ255) [reply tests/test_text_to_num_ru.py]: fixed

**Comment by rtxm** [comment]
LGTM

**Pull request merged**